faq/edge-services.mdx

```diff
@@ -10,4 +10,20 @@ dates:
 category: network
 ---
 
-TODO ADD HERO
\ No newline at end of file
+TODO ADD CORRECT HERO
+
+## What is Edge Services?
+
+Edge Services is an feature for Scaleway Load Balancers and Object Storage buckets, currently available in [Public Beta](https://www.scaleway.com/en/betas/). It provides a [caching service](/network/edge-services/how-to/configure-cache/) to improve performance by reducing load on your [origin](/network/edge-services/concepts/#origin), and a customizable and secure endpoint for accessing content via Edge Services, which can be set to a subdomain of your choice.
```

Comment:
Can it be only a subdomain or can I use it also on my primary domain ( cache.example.com vs exaple.com ) ?

Reply:
It has to be a subdomain, can't be a primary domain. This is specified in the main doc, do you think it should be said here too?

Reply:
no, it is fine then. I was just not sure if it is possible 👍